tests/unit/amo/api/test_index.js

```diff
@@ -681,9 +681,13 @@ describe(__filename, () => {
   });
 
   describe('startLoginUrl', () => {
-    const getStartLoginQs = ({ _config, location }) => {
+    const getStartLoginQs = ({
+      _config,
+      _window,
+      location = createFakeLocation(),
```

Comment:
nit: I'm not sure I'd have bothered creating a default value for `location` when only one test calls `getStartLoginQs` without providing it - it could have just been specified in `'uses window.location when running locally'`